frontend/src/app/containers/Restart.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
For the sake of having an easy rule, I'd suggest putting anything that gets handed to the router in `containers`, and anything else in `components`.